maintainers/maintainer-list.nix

```
@@ -10404,4 +10404,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  holgerpeters = {
+    name = "Holger Peters";
+    email = "holger.peters@posteo.de";
+    github = "HolgerPeters";
+    githubId = 4097049;
+  }
```

```suggestion
  };
```

Please keep this file alphabetically sorted.